pdns/dbdnsseckeeper.cc

```diff
@@ -589,6 +589,26 @@ bool DNSSECKeeper::getTSIGForAccess(const DNSName& zone, const string& master, D
   return false;
 }
 
+bool DNSSECKeeper::unSecureZone(const DNSName& zone, string& error, string& info) {
+  // Not calling isSecuredZone(), as it will return false for zones with zero
+  // active keys.
+  DNSSECKeeper::keyset_t keyset=getKeys(zone);
```

Comment:
Just a nit, but perhaps `isSecureZone()` could get an optional boolean to return `true` as long as there is one key, be it inactive?

Reply:
That might make sense, perhaps turn that into a ticket?